pyiron/atomistics/job/sqs.py

```diff
@@ -148,10 +150,30 @@ def from_hdf(self, hdf=None, group_name=None):
             hdf=hdf,
             group_name=group_name
         )
-        with self.project_hdf5.open("input") as h5in:
-            self.input.from_hdf(h5in)
+        self._backwards_compatible_input_from_hdf()
         with self.project_hdf5.open("output/structures") as hdf5_output:
             structure_names = hdf5_output.list_groups()
         for group in structure_names:
             with self.project_hdf5.open("output/structures/" + group) as hdf5_output:
                 self._lst_of_struct.append(Atoms().from_hdf(hdf5_output))
+
+    def _backwards_compatible_input_from_hdf(self):
+        if "HDF_VERSION" in self.project_hdf5.list_nodes():
+            version = self.project_hdf5["HDF_VERSION"]
+        else:
+            version = "0.1.0"
+
+        if version == "0.1.0":
+            with self.project_hdf5.open("input") as hdf5_input:
+                try:
+                    gp = GenericParameters(table_name="custom_dict")
+                    gp.from_hdf(hdf5_input)
+                    for k in gp.keys():
+                        self.input[k] = gp[k]
+                except TypeError:
```

Comment:
This is only in the ScriptJob because of the script_path member there, so we don't need it here.

Reply:
"this" here means the `try/except` right?